packages/stream-model-handler/src/model-handler.ts

```diff
@@ -137,70 +136,14 @@ export class ModelHandler implements StreamHandler<Model> {
       log: [{ cid: commitData.cid, type: CommitType.GENESIS }],
     }
 
```

Comment:
Recommend adding `Model.assertComplete(payload.data)` before the call to `assertNoExtraKeys` on line 120